src/convertToObject.js

```diff
@@ -12,7 +12,22 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const styleObj = {};
+
+  const sourseArray = sourceString
+    .split(';')
+    .map(item => item.trim())
+    .filter(item => item.length !== 0);
+
+  sourseArray.forEach(item => {
+    const rules = item.split(':');
+    const property = rules[0].trim();
+    const value = rules[1].trim();
+
+    styleObj[property] = value;
```

Comment:
не часто сам таке роблю але якщо вже виправлятимеш назви то для красоти можна:
```suggestion
  sourseArray.forEach(item => {
    let { key, value } = item.split(':');
    key = key.trim();
    value = value.trim();

    styleObj[key] = value;
```
Хоча міг трохи помилитися з синтаксисом без едітора але переприсвоєння має спрацювати без лишніх змінних..

Reply:
я їх називала згідно того, яквони називаються у Цсс, але сама не дуже впевнена, що пам"ятаю їх правильно, тому краще перейменую до ключів, значень та властивостей, напевно